libraries/WiFi/examples/WiFiScan/WiFiScan.ino

```diff
@@ -19,30 +19,69 @@ void setup()
 
 void loop()
 {
-    Serial.println("scan start");
+    Serial.println("Scan start");
 
-    // WiFi.scanNetworks will return the number of networks found
+    // WiFi.scanNetworks will return the number of networks found.
     int n = WiFi.scanNetworks();
-    Serial.println("scan done");
+    Serial.println("Scan done");
     if (n == 0) {
         Serial.println("no networks found");
     } else {
         Serial.print(n);
         Serial.println(" networks found");
+        Serial.println("Nr | SSID            | RSSI | CH | Encryption");
         for (int i = 0; i < n; ++i) {
             // Print SSID and RSSI for each network found
-            Serial.print(i + 1);
-            Serial.print(": ");
-            Serial.print(WiFi.SSID(i));
-            Serial.print(" (");
-            Serial.print(WiFi.RSSI(i));
-            Serial.print(")");
-            Serial.println((WiFi.encryptionType(i) == WIFI_AUTH_OPEN)?" ":"*");
+            Serial.printf("%2d",i + 1);
+            Serial.print(" | ");
+            if (WiFi.SSID(i).length() < 15) {
```

Comment:
Suggest changing to accommodate maximum 32 character SSID to:
`if ( WiFi.SSID( i ).length() < 32 ) {`

Reply:
The limitation was on purpose because it is easier to read a table row without a huge space between the columns.
Most SSIDs were shorter than 15 chars on my test setup.